e2e/tests/ui-driven/src/create-flow-with-geospatial.spec.ts

```diff
@@ -169,7 +176,51 @@ test.describe("Flow creation, publish and preview", () => {
     ).toBeVisible();
     await clickContinue({ page });
 
+    const drawBoundaryTitle = page.getByRole("heading", {
+      name: "Confirm your location plan",
+    });
+    await expect(
+      drawBoundaryTitle,
+      "We are in the Draw Boundary component",
+    ).toBeVisible();
```

Comment:
nit: `await expect(drawBoundaryTitle).toBeVisible();` is a very clear indicator that we're on the right page to me and I'm not sure I really understand the value of of the second `expect` argument here - feels quite noisy?